app/redux/smesher/reducer.ts

```diff
@@ -150,6 +155,16 @@ const reducer = (state: SmesherState = initialState, action: CustomAction) => {
     }
     case LOGOUT:
       return initialState;
+    case SET_METADATA: {
+      const {
+        payload: { smeshingStart = 0, posInitStart = 0 },
```

Comment:
We may not need such defaults here.
Otherwise, we may end up with overwriting the correct timestamp with "January 1st, 1970".